modules/react/_examples/stories/examples/CookieBanner.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+
+import {createComponent, styled, StyledType} from '@workday/canvas-kit-react/common';
+import {colors, commonColors, depth, type, space} from '@workday/canvas-kit-react/tokens';
+import {Hyperlink, HyperlinkProps, PrimaryButton} from '@workday/canvas-kit-react/button';
+
+const CookieBannerItem = createComponent('div')({
+  displayName: 'CookieBanner.Item',
+  Component: ({isRow, ...elProps}: ItemProps, ref) => (
+    <BannerItem ref={ref} isRow={isRow} {...elProps} />
+  ),
+});
+
+const CookieBannerSettings = createComponent('a')({
+  displayName: 'CookieBanner.Settings',
+  Component: (props: CookieSettingsProps, ref, Element) => (
+    <CookieSettings ref={ref} as={Element} {...props} />
+  ),
+});
+
+const CookieBanner = createComponent('div')({
+  displayName: 'CookieBanner',
+  Component: (props: CookieBannerProps, ref, Element) => (
+    <Banner ref={ref} as={Element} {...props} />
+  ),
+  subComponents: {Item: CookieBannerItem, Settings: CookieBannerSettings},
+});
+
+export const Basic = () => {
+  const DefaultNotice = `We use cookies to ensure that we give you the best experience on our website. 
+    If you continue without changing your settings, we'll assume that you are willing to receive cookies.`;
+  const onClickSettings = e => e.preventDefault();
+
+  return (
+    <ExampleContainer>
+      <CookieBanner isClosed={false}>
+        <CookieBanner.Item>{DefaultNotice}</CookieBanner.Item>
+        <CookieBanner.Item isRow>
+          <CookieBanner.Settings onClick={onClickSettings}>Settings</CookieBanner.Settings>
```

Comment:
I think we can just render a `tertiary button here`

Reply:
Thanks for catching this. I agree, it's better to put `TertiaryButton` here.